src/blog/templates/blog_post.html

```diff
@@ -83,9 +84,19 @@ <h2 class="text-center">{{object.title}}</h2>
           </p>
           <ul class="pager">
                 <li>
-                  <a href="{% url "blog_index" %}">&larr; Cuprins</a>
+                  <a href="{% url 'blog_index' %}">&larr; Cuprins</a>
                 </li>
           </ul><!-- ./pager -->
+          <hr>
+          <h2 style="margin-left:10px" class="intro-text text-center">
```

Comment:
Ar fi cool daca vrei sa fi perfectionist sa folosesti clasele de aliniere bootstrap 3, sau daca nu sunt (uneori nu face ce vrei) sa incluzi CSS-ul in fisierul `custom.css` care cred ca e pe undeva in static.
Da intrucat o sa luam tema, si tema o sa aiba alt css, si tag-uri presupun si formatare proprie, atunci putem lasa si asa deocamdata.

Ideea este tot consistenta, toate customizarile css ale tale in plus fata de boostrap sa fie in `custom.css` sau ceva similar.

Reply:
https://github.com/selfpower-ngo/selfpower.ro/commit/42a4505ca513c1bcaa86a8b1392d14d4d5b859a8
_Ar fi cool daca vrei sa fi perfectionist sa folosesti clasele de aliniere bootstrap 3, sau daca nu sunt (uneori nu face ce vrei) sa incluzi CSS-ul in fisierul custom.css care cred ca e pe undeva in static._
**N am gasit totusi static-ul : Exista doar un director static_dirs si nu schimba deloc styling ul cand faceam o clasa css ca in commit ul de sus: span-spacing**

Reply:
Rezolvat cu ce mi-ai zis:
Step 1:
https://github.com/selfpower-ngo/selfpower.ro/commit/ed0c90f10ad0d01c0e8b7abde9adef0e7c9b8202
Step 2:
`python src\manage.py collectstatic`
Step 3: 
Folosind **incognito mode** sau alte metode de a evita _cache ul deja existent_ in browser